### PR TITLE
Proposal to change license to GPLv3

### PR DESCRIPTION
**Pull request by karthikeyan5**
Please approve this license change.

### CODE REVIEW

File: license.txt [hunk 1]
```diff
@@ -1 +1,674 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (Vigneshsekar) [inline license.txt:634]: Have to fill it here

File: license.txt [hunk 2]
```diff
@@ -1 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (Vigneshsekar) [inline license.txt:655]: Here too?

File: license.txt (hunk 1 above)
Comment (karthikeyan5) [reply license.txt:634]: nope, not required... it actually gives you a template of what to include in each file... and you are not expected to make any changes to the license.txt file...

File: license.txt (hunk 2 above)
Comment (karthikeyan5) [reply license.txt:655]: same as above... no changes required...

**Review by KaviyaPeriyasamy — APPROVED** [review]

**Review by Vigneshsekar — APPROVED** [review]

**Review by M0hammedHaris — APPROVED** [review]

**Review by saiindhu — APPROVED** [review]

**Pull request merged**